packages/be/liftoff.js

```diff
@@ -53,7 +53,7 @@ try {
 // -----------------------------------------------------------------------------
 // const users = async () => {
 //   try {
-//     // const user = await MC.model.User.findOne({ githubUsername: 'timelytree' })
+//     const user = await MC.model.User.findOne({ githubUsername: 'timelytree' })
```

Comment:
@timelytree actually I don't want to merge this because I think it might be in your testing state, based on the branch it was forked from, right?

Reply:
These commented out functions in `liftoff.js` are perpetually there, including in production, all good 😊